test/unit/specs/components/staking/__snapshots__/LiDelegate.spec.js.snap

```diff
@@ -28,7 +28,7 @@ exports[`LiDelegate has the expected html structure 1`] = `
       class="li-delegate__value number_of_votes num bar"
     >
       <span>
-        14
+        14.00
```

Comment:
Are we sure on always showing fractions?

Reply:
the goal is to show the most accurate way possible the amount of tokens that a candidate has. The explorer already shows fractions (specially in the ones that got slashed)

Reply:
OK